erpnext/stock/doctype/stock_reconciliation_item/stock_reconciliation_item.json

```diff
@@ -166,10 +167,18 @@
    "fieldtype": "Link",
    "label": "Batch No",
    "options": "Batch"
+  },
+  {
+   "default": "0",
+   "fieldname": "allow_zero_valuation_rate",
+   "fieldtype": "Check",
+   "label": "Allow Zero Valuation Rate",
+   "print_hide": 1
```

Comment:
Let's keep this read only and have its display depend on its own value. It is auto set for customer provided items anyway. And as of now for regular items we don't need this.